src/components/Header/Header.tsx

```diff
@@ -1,18 +1,37 @@
-import React from "react";
+import React, { useState } from "react";
 import chatImg from "../../static/svg/chatLogo.png";
 import { Menu } from "./Menu";
 import { ImageHeader } from "./ImageHeader";
 import { Navigation } from "./Navigation";
 import { UserProfile } from "./UserProfile";
+import { HamburgerMenu } from "./HamburgerMenu";
+import times from "../../static/svg/times.svg";
 
-export const Header = () => (
-  <div>
-    <div className="flex justify-between items-center content-center h-16 lg:h-12 flex justify-between items-center content-center w-full">
-      <Menu />
-      <ImageHeader src={chatImg} width={73} className="h-6 lg:ml-12" />
-      <Navigation />
-      <UserProfile />
+export const Header = () => {
+  const [openMenu, setOpenMenu] = useState(false);
+  return (
+    <div>
+      {openMenu === true && (
+        <div>
+          <div className="flex justify-end mr-6 mt-4">
+            <div onClick={() => setOpenMenu(false)}>
+              <ImageHeader src={times} />
+            </div>
+          </div>
+          <HamburgerMenu />
+          <div
+            className="opacity-75 bg-darkblue h-full w-full lg:hidden"
+            onClick={() => setOpenMenu(false)}
+          />
+        </div>
+      )}
+      <div className="flex justify-between items-center content-center h-16 lg:h-12 w-full">
+        <Menu onClick={() => setOpenMenu(true)} />
```

Comment:
as I tried this in browser, I would like to have pointer cursor and some hover effect when I hover over this icon from a design standpoint